recipes/micromamba/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: micromamba
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/mamba/archive/{{ version }}.tar.gz
+  sha256: 2c1029d350be219c682ea803004cadd4dc40c9a5029784769684530c9a17c0ef
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  ignore_run_exports:
+    - libcurl
+    - libarchive
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host:
+    - cli11
+    - nlohmann_json
+    # static libs
+    - libcurl
+    - libarchive
+    - krb5-static
+    - libsolv-static
+    - libopenssl-static
+
+test:
+  commands:
+    - test -f $PREFIX/bin/micromamba  # [unix]
+    - micromamba --help
+
+about:
+  home: https://github.com/TheSnakePit/mamba
+  license_file:
+    - LICENSE
+    - CURL_LICENSE.txt
+    - LIBARCHIVE_LICENSE.txt
+    - KRB5_LICENSE.txt
+    - LIBSOLV_LICENSE.txt
+    - LIBOPENSSL_LICENSE.txt
```

Comment:
What about cli11 and nlohmann_json?

Reply:
they aren't statically linked but rather header-only.

Reply:
I haven't seen anyone do something with regards to header only libraries. The symbols are not exported, so they can't be reused as far as I understand.

Reply:
Done -- @isuruf said they should be added so they're added now.